app/models/events/pull_request.rb

```diff
@@ -61,17 +63,22 @@ def opened(payload)
         PullRequestJobs::Opened.perform_later(payload)
       end
 
-      def closed(payload)
-        PullRequestJobs::Closed.perform_later(payload)
-      end
+    # Actions
 
-      def review_requested(payload)
-        PullRequestJobs::ReviewRequested.perform_later(payload)
-      end
+    def open
+      open!
+      update!(opened_at: Time.current)
+    end
 
-      def review_request_removed(payload)
-        PullRequestJobs::ReviewRequestRemoved.perform_later(payload)
-      end
+    def merged
+      update!(merged_at: Time.current)
+    end
+
+    def closed
```

Comment:
The name of the methods is the same as the name of the action that we receive from github, another way to handle this would be by change ACTIONS to a hash where it maps the name of the action as the key and our method as the value.

Reply:
Actually this file should be discarded from this pr, this was solved in this [pr](https://github.com/rootstrap/rs-code-review-metrics/pull/41)